src/api/mod.rs

```diff
@@ -34,24 +26,14 @@ pub use typed::{
 };
 
 mod resource;
-pub use self::resource::{
-    Object,
-    ObjectList,
-    WatchEvent,
-    KubeObject,
-};
+pub use self::resource::{KubeObject, Object, ObjectList, WatchEvent};
 
 #[cfg(feature = "openapi")]
 mod openapi;
 #[cfg(feature = "openapi")]
 mod snowflake;
 #[cfg(feature = "openapi")]
-pub use snowflake::{v1Event, v1Secret, v1ConfigMap};
+pub use snowflake::{v1ConfigMap, v1Event, v1Secret};
 
 mod metadata;
-pub use self::metadata::{
```

Comment:
I had to add `ListMeta` to the public exports for use in the example in the docs on the new `iter` methods, is this okay or should I just cut out the examples?

Reply:
that should probably be public anyway :+1:

thanks for pointing it out though. the re-formatting made that hard to spot.